ios/brave-ios/Sources/Brave/Frontend/Browser/Handlers/AboutHomeHandler.swift

```diff
@@ -14,14 +14,24 @@ public class AboutHomeHandler: InternalSchemeResponse {
   public func response(forRequest request: URLRequest) async -> (URLResponse, Data)? {
     guard let url = request.url else { return nil }
     let response = InternalSchemeHandler.response(forUrl: url)
-    let bg = UIColor.braveBackground.toHexString()
+
+    let lightModeColour = UIColor.braveBackground.resolvedColor(
+      with: .init(userInterfaceStyle: .light)
+    ).toHexString()
+    let darkModeColour = UIColor.braveBackground.resolvedColor(
```

Comment:
We should be using `UIColor(braveSystemName: .containerBackground)` to match what we set on the web view background in night mode

Reply:
Ahh will change one sec. I didn't realize it was different since the old code was using braveBackground :o